Components/ConnectFactory.php

```diff
@@ -201,7 +201,8 @@ public function createSDK()
             new DefaultCategoryResolver(
                 $manager,
                 $manager->getRepository('Shopware\CustomModels\Connect\RemoteCategory'),
-                $manager->getRepository('Shopware\CustomModels\Connect\ProductToRemoteCategory')
+                $manager->getRepository('Shopware\CustomModels\Connect\ProductToRemoteCategory'),
+                $manager->getRepository('Shopware\Models\Category\Category')
```

Comment:
Please replace Shopware\CustomModels\Connect\ProductToRemoteCategory and Shopware\Models\Category\Category with ProductToRemoteCategory::class and Category::class